tasks/time-series/time-series-forecasting/MLproject

```diff
@@ -0,0 +1,14 @@
+name: time-series-forecasting
+
+conda_env: conda.yaml
+
+entry_points:  
+  main:
+    parameters:
+      city: {type: string, default: 'Hamburg'}
+    command: "python main.py {city}"
+
+  train_weathernet:
```

Comment:
If you choose to keep only one python script then please remove it from MLproject or even better, put both the scripts back.